content/concepts/transport.md

```diff
@@ -210,3 +210,76 @@ Following the multiaddress format described earlier, a standard QUIC connection
 look like: `/ip4/127.0.0.1/udp/65432/quic/`.
 
 In this section, we offered an overview of QUIC and how QUIC works in libp2p.
+
+## WebTransport
+
+Another transport protocol under development at the IETF is WebTransport.
+WebTransport is a new specification that uses QUIC to offer an alternative to
+WebSockets. It can be considered WebSockets over QUIC by allowing 
+browsers to establish a stream-multiplexed and bidirectional 
+connection to servers. 
+
+The specification can depend on and reuse the QUIC infrastructure in place 
+to offer WebSockets that take the benefits of UDP and offer streams without 
+head-of-line blocking.
+
+Recall that WebSockets are bidirectional, full-duplex communication between two 
+points over a single-socket connection. WebTransport can be used
+like WebSockets, but with the support of multiple streams.
+
+WebTransport supports reliable streams that can be arbitrary in size. They can be 
+independent and canceled if needed. The datagrams in a 
+WebTransport connections are MTU-sized.
```

Comment:
```suggestion
```

Instead, we could say that WebTransport therefore exhibits all the advantages of QUIC over TCP (faster handshake, no HoL blocking, future-proof).